sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/finetuning/finetuning_job.py

```diff
@@ -0,0 +1,126 @@
+from azure.ai.ml.entities._job.job import Job
+from azure.ai.ml.entities._job.job_io_mixin import JobIOMixin
+from typing import Any, Dict
+from azure.ai.ml._restclient.v2024_01_01_preview.models import (
+    ModelProvider as RestModelProvider,
+    JobBase as RestJobBase,
+)
+from azure.ai.ml._utils.utils import camel_to_snake
+from azure.ai.ml.exceptions import ErrorCategory, ErrorTarget, ValidationException
+from azure.ai.ml.constants._job.finetuning import FineTuningConstants
+
+
+class FineTuningJob(Job, JobIOMixin):
```

Comment:
Please add @eperiment annotation to these new classes. https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=/docs/SDK_preview_features.md&_a=preview&anchor=%40experimental-usage-examples